src/tiles/expandable-tile.component.ts

```diff
@@ -22,11 +22,11 @@ export interface ExpandableTileTranslations {
 			type="button"
 			(click)="onClick()">
 			<div [attr.aria-label]="(expanded ? collapse : expand).subject | async" class="bx--tile__chevron">
```

Comment:
Do we need the `aria-label` here if we have a `title` on the SVG? Seems like it may be redundant in this case

Reply:
The attr.aria-label was not changed and was left the way it is

Reply:
However, divs should not have aria label attributes since they aren't picked by the screen readers anyway

Reply:
Gotcha, we should drop it then 👍 No need to have excess attributes potentially messing things up